docker/application.properties

```diff
@@ -21,7 +21,7 @@ git.github.repo=http://github.com/org/repo
 # Database Properties
 spring.datasource.url=jdbc:postgresql://postgres:5432/test-reporter
 spring.datasource.username=postgres
-#spring.datasource.password=testreporter
+spring.datasource.password=testreporter
```

Comment:
Are you sure this is necessary and will work by default? 
If I'm correct, by default, there is no password on postgres user